app/lib/gitgithub/IssueUpdater.scala

```diff
@@ -35,7 +37,7 @@ trait IssueUpdater[IssueType <: PullRequest, PersistableState, Snapshot <: State
   def process(issueLike: IssueType): Future[Option[Snapshot]] = {
     logger.trace(s"handling ${issueLike.prId.slug}")
     for {
-      oldLabels <- new LabelledState(issueLike, _ => true).currentLabelsF
+      oldLabels <- new LabelledState(issueLike, repoSnapshot.allPossibleCheckpointPRLabels).currentLabelsF
```

Comment:
`LabelledState` constructor seems to take a `String => Boolean` as second argument. How does this line work then?